src/doc/src/reference/workspaces.md

```diff
@@ -57,6 +59,14 @@ version = "0.1.0"    # the current version, obeying semver
 authors = ["Alice <a@example.com>", "Bob <b@example.com>"]
 ```
 
+In the [worksapce section](#the-workspace-section), you can specify the members of 
+the workspace by [workspace.members filed](#the-members-and-exclude-fields). 
+
+If no other members are specified in [workspace.members filed](#the-members-and-exclude-fields) 
+in `Cargo.toml` in the workspace root directory, the root package will 
+become the default workspace member. In addition, you can also specify the default 
+workspace members through [workspace.default-members](#the-default-members-field).
```

Comment:
Most of this addition seems a bit redundant, since some points are already covered in other parts of this doc. Perhaps we could do something like:

```suggestion
Unless explicitly set through [`workspace.default-members`](#the-default-members-field),
the root package is considered as the only default member of the workspace.
```